src/components/form-elements/Checkbox.jsx

```diff
@@ -42,7 +42,7 @@ class Checkbox extends PureComponent<CheckboxPropsType, CheckboxStateType> {
           {...additionalProps}
         />
         <label className="sg-checkbox__ghost" htmlFor={id}>
-          <Icon type={TYPE.CHECK} color={ICON_COLOR.ADAPTIVE} size={10} />
+          <Icon type="std-check" color={ICON_COLOR.ADAPTIVE} size={16} />
```

Comment:
maybe we could do just `color="adaptive"` here as well?

Reply:
done